spotbugs/src/main/java/edu/umd/cs/findbugs/xml/XMLUtil.java

```diff
@@ -33,4 +38,28 @@ public static <T> List<T> selectNodes(Node node, String arg0) {
         return (List<T>) node.selectNodes(arg0);
     }
 
+    public static SAXReader buildSAXReader() {
+        SAXReader reader = new SAXReader();
+
+        try {
+            reader.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
+        } catch (Exception e) {
+            throw new IllegalArgumentException(e);
```

Comment:
It seems that it's not argument issue (this `buildSAXReader()` method has no argument).
It is probably better to use another exception type?

Reply:
I  have modified the exception type and added an error message, please let me know if you had something else in mind or if this requires further adjustments